### DISCUSSION TITLE
How to call get_ beta_ build_ metrics ？

### DISCUSSION THREAD

**Discussion by csc-EricWu** [fastlane/fastlane#17552] [Q&A]
https://github.com/fastlane/fastlane/blob/feb8cc09c9976f7f460203cf9486fd28d31f6955/spaceship/spec/connect_api/models/beta_build_metric_spec.rb

**Comment by csc-EricWu** [comment]
I want to know the total number of external installations of build.

**Comment by rogerluan** [comment]
I don't understand the issue, since you pointed out to the code that shows exactly how to get that info 😅 
You need to implement this in ruby (e.g. `Fastfile` would do): `response = Spaceship::ConnectAPI.get_beta_build_metrics` and then parse the response just like shown in the specs you posted above.

**Reply by csc-EricWu** [reply]
Thank you for your reply. When I execute, I get the URL path is not valid - the resource 'beta buildmetrics' does not exist

**Reply by rogerluan** [reply]
Oh, in that case, that's a problem 😅 Can you file a bug report here? https://github.com/fastlane/fastlane/issues/new/choose
Appreciate it!
